context-object/src/main/java/com/iluwatar/context/object/ServiceContext.java

```diff
@@ -0,0 +1,35 @@
+package com.iluwatar.context.object;
+
+/**
+ * Where context objects are defined.
+ */
+public class ServiceContext {
+
+    String ACCOUNT_SERVICE, SESSION_SERVICE, SEARCH_SERVICE;
+
+    public void setACCOUNT_SERVICE(String ACCOUNT_SERVICE) {
+        this.ACCOUNT_SERVICE = ACCOUNT_SERVICE;
+    }
+
+    public void setSESSION_SERVICE(String SESSION_SERVICE) {
+        this.SESSION_SERVICE = SESSION_SERVICE;
+    }
+
+    public void setSEARCH_SERVICE(String SEARCH_SERVICE) {
+        this.SEARCH_SERVICE = SEARCH_SERVICE;
+    }
+
+    public String getACCOUNT_SERVICE() {
+        return ACCOUNT_SERVICE;
+    }
+
+    public String getSESSION_SERVICE() {
+        return SESSION_SERVICE;
+    }
+
+    public String getSEARCH_SERVICE() {
+        return SEARCH_SERVICE;
+    }
+
+    public String toString() { return ACCOUNT_SERVICE + " " + SESSION_SERVICE + " " + SEARCH_SERVICE;}
```

Comment:
- Using capital letters in method names looks odd
- It's better to use Lombok for getters/setters and other boilerplate
- I think Lombok can also handle toString